packages/falter-berlin-bbbdigger/files/postinst.sh

```diff
@@ -52,12 +52,11 @@ uci set network.${IFACE}_dev.name=$IFACE
 
 uci set network.$IFACE=interface
 uci set network.$IFACE.proto=dhcp
-uci set network.$IFACE.ifname=$IFACE
+uci set network.$IFACE.device=$IFACE
 
 # firewall setup (first remove from the zone and add it back)
-ZONE=$(echo $(uci show firewall.zone_freifunk.network | cut -d \' -f 2 | sed "s/${IFACE}//g"))
-ZONE="$ZONE $IFACE"
-uci set firewall.zone_freifunk.network="${ZONE}"
+uci -q del_list firewall.zone_freifunk.network=$IFACE
```

Comment:
Can you explain this change further please? What's exactly happening here?

Reply:
since people can accidentally reinstall the package, we make sure that the firewall is set up properly.  If you look above in the code, we also make sure that the list of servers is up to date.